layouts/publications/list.html

```diff
@@ -0,0 +1,261 @@
+            if (data.length) {
+                const tableDiv = document.createElement("div");
+                tableDiv.id = tableDivID;
+                tableDiv.classList.add("pub-table", "active");
+
+                const textDiv = document.createElement("div");
+                textDiv.classList.add("pb-3");
+                textDiv.innerHTML = `The following are 10 most recent publications for <code style="text-transform: capitalize">
+                                    ${pubCat}</code>, to see all the available publications visit 
+                                    <a target="_bank" href="${getQueryURL(pubCat)}">Europe PMC</a>`;
+                tableDiv.appendChild(textDiv);
+
+                const table = document.createElement("table");
+                table.id = tableID;
+                table.classList.add("table", "table-hover");
+
+                const thead = document.createElement("thead");
+                thead.classList.add("table-light");
+                const tbody = document.createElement("tbody");
+                const headers = ["Publications"];
+                const headerRow = document.createElement("tr");
+
+                headers.forEach(header => {
+                    const th = document.createElement("th");
+                    th.textContent = header;
+                    th.scope = "col";
+                    headerRow.appendChild(th);
+                });
+                thead.appendChild(headerRow);
+
+                data.forEach(item => {
+                    const row = document.createElement("tr");
+                    const publication = document.createElement("td");
+                    const title = `<div><a href="https://doi.org/${item.doi}" target="_blank"><span style="font-weight:500;">${item.title}</span></a></div>`;
+                    const authors = `<div><span class="text-muted" style="font-size:0.82em">${item.authorString}</span></div>`;
+                    const journal = `<div><span class="text-muted" style="font-size:0.82em">${item.journalInfo?.journal?.title}</span></div>`;
+                    const doi = `<div><span class="text-muted" style="font-size:0.82em">DOI: ${item.doi}</span></div>`;
+
+                    publication.innerHTML = title + authors + journal + doi;
+                    row.appendChild(publication);
+                    tbody.appendChild(row);
+                });
+
+                table.appendChild(thead);
+                table.appendChild(tbody);
+
+                tableDiv.appendChild(table)
+                pubContainer.appendChild(tableDiv);
+            } else {
+                const noPubText = document.createElement("p");
+                noPubText.id = "no-pub-text";
+                noPubText.style.textAlign = "center";
+                noPubText.innerHTML = `No publications were retireved for <code style="text-transform: capitalize">
+                                        ${pubCat}</code>, notify portal team.`;
```

Comment:
@LianeHughes and here _(line 173 - 174)_, if something is wrong and could not fetch any publications it displays

<img width="700" height="159" alt="Screenshot 2025-08-21 at 11 43 52" src="https://github.com/user-attachments/assets/48a68bae-c491-4dc2-b56c-760fbd37afbb" />